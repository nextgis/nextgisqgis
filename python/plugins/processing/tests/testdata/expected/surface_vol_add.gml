<?xml version="1.0" encoding="utf-8" ?>
<ogr:FeatureCollection
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://ogr.maptools.org/ surface_vol_add.xsd"
     xmlns:ogr="http://ogr.maptools.org/"
     xmlns:gml="http://www.opengis.net/gml">
  <gml:boundedBy><gml:null>missing</gml:null></gml:boundedBy>
                                                                                                                                                                                                                                                                                                
  <gml:featureMember>
    <ogr:surface_vol_add fid="surface_vol_add.0">
      <ogr:volume>0.06802753</ogr:volume>
      <ogr:deg2>0.00130550</ogr:deg2>
      <ogr:pixel_count>130550</ogr:pixel_count>
    </ogr:surface_vol_add>
  </gml:featureMember>
</ogr:FeatureCollection>
