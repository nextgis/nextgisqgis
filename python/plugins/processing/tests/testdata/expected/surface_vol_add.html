<html><head><meta http-equiv="Content-Type" content="text/html;charset=utf-8"/></head><body>
<p>Analyzed file: /home/nyall/dev/QGIS/python/plugins/processing/tests/testdata/dem.tif (band 1)</p>
<p>Volume: 0.06802752691184032</p>
<p>Pixel count: 130550</p>
<p>Area: 0.001305499999999814 deg2</p>
</body></html>